data/login.js

```diff
@@ -34,10 +34,20 @@ const logInFunctions = {
     if (validations.validateIsString([emailAddress, password]) === 0) {
       throw "Error : All inputs must be valid String";
     }
-
     emailAddress = emailAddress.trim().toLowerCase();
     password = password;
 
+    const userCollection = await users();
+    const ifAlready = await userCollection.findOne(
+      { email: emailAddress },
```

Comment:
I removed this in my last pull request ! Now If you try to login with the new not registered user it will, show the server error.

Please confirm if I am right.